doc/admin-guide/files/ssl_server_name.config.en.rst

```diff
@@ -113,13 +112,13 @@ Disable HTTP/2 for ``no-http2.example.com``.
 
    server_config = {{ fqdn="no-http2.example.com", disable_h2=true }}
 
-Require client certificate verification for ``example.com`` and ``yahoo.com``.
+Require client certificate verification for ``example.com`` and any servername ending with ``*.yahoo.com``. So client request for any server whose name ends with yahoo.com (for e.g., def.yahoo.com, abc.yahoo.com etc.) will make ATS require and verify the client certificate. Again, ATS will ask for client certficate for ``example.com`` but will only strictly verify the certificate if the client provides one.
```

Comment:
Use `|TS|`, not `ATS`. `server name`, not `servername`. "Ending with ``.yahoo.com``.  Don't start sentences with "So". Perhaps "therefore" or "As a result". "A client requst for a server name ending in 'yahoo.com`" ...

"Will make ATS" -> "will cause |TS| to". "For example.com, by contrast, |TS| will allow a client certificate to be provided and if it is, |TS| will require the certificate to be valid".